.github/workflows/ci.yml

```
@@ -0,0 +1,62 @@
+name: flake8-async CI
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
```

```suggestion
    - uses: actions/checkout@v3
```
I'll update post merge